src/ProjectTemplates/test/e2eTestSettings.ci.json

```diff
@@ -4,5 +4,7 @@
   "DefaultWaitTimeoutInSeconds": 120,
   // This value is balanced between completing the build fast enough upon failure and giving
   // each E2E test a fair chance to pass even in the event that a separate test has failed already.
-  "DefaultAfterFailureWaitTimeoutInSeconds": 120
+  "DefaultAfterFailureWaitTimeoutInSeconds": 120,
+  // Capture all the logs from the browser when an exception occurs.
+  "BrowserLogLevel": "All"
```

Comment:
Do we really need this to be configurable? We wouldn't `All` be the only value we ever care about?

Reply:
It seems like if anything `All` should be the default.

Reply:
I might take this out based on the assertion consolidation that is going on on the Blazor infrastructure cleanup PR. I wanted to have an option where we could leave Blazor infrastructure as it is instead of forcing the same value on both test harnesses, but I think its fair to use All for both of them.

As I said my main motivation was to avoid forcing changes on Blazor, but given your feedback I'll make them.

Reply:
You say *force* - I say *improve*